ZenPacks/zenoss/Microsoft/Windows/migrate/MigrateRRDPaths.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+"""Remove Windows services.
+ZPS-1204
+Prevent data loss due to rrdPath changes between ZPL and non-ZPL-based versions
+of this zenpack
+
+"""
+
+# Logging
+import logging
+
+# Zenoss Imports
+from Products.ZenModel.migrate.Migrate import Version
+from Products.ZenModel.ZenPack import ZenPackMigration
+
+LOG = logging.getLogger('zen.MicrosoftWindows')
+
+
+class MigrateRRDPaths(ZenPackMigration):
+    version = Version(2, 7, 0)
+
+    def migrate(self, pack):
+        if pack.prevZenPackVersion is None:
+            return
+        installed_version = Version.parse('Zenoss {}'.format(pack.prevZenPackVersion))
```

Comment:
Always prefer `pkg_resources.parse_version` to `Version.parse` for new code. It's usage is the same.

```python
installed_version = pkg_resources.parse_version(pack.prevZenPackVersion)
if installed_version < pkg_resources.parse_version('2.6.0'):
```